lib/vsc/utils/docs.py

```diff
@@ -57,7 +57,7 @@ def mk_rst_table(titles, columns):
         width = max(map(len, columns[i] + [title]))
 
         # make line template
-        tmpl.append('{%s:{c}<%s}' % (i, width))
+        tmpl.append('{{{}:{{c}}<{}}}'.format(i, width))
```

Comment:
what is this unreadable crap?

Reply:
to escape a `{` you need to double it.... I'll see if I can come up with some more readable.

Reply:
rewritten with ljust builtin instead of format on format on ...